<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_mainwindow</class>
 <widget class="QMainWindow" name="login_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control de Acceso</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>NOVALIB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alias</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alias_lineedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrasena_lineedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="acceder_pushbutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceder</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
